docs/API.md

```diff
@@ -2010,6 +2011,31 @@ mp.on('modAddDj', (data) => {
 })
 ```
 
+<a id="#event-modban"></a>
+## 'modBan'
+
+Fired when a moderator bans a user.
+
+**Parameters**
+
+ - `moderator` - The staff [User](#class-user) who banned the user.
+ - `username` - The name of the user that was banned.
+ - `duration` - The duration of time the ban lasts for
+
+```js
+mp.on('modAddDj', (data) => {
```

Comment:
I need to improve my cp skillz :/

I'll wait to change this, since I'll probably have to change something else due to travis failing.